mobile/lib/shared/ui/immich_app_bar.dart

```diff
@@ -80,6 +81,7 @@ class ImmichAppBar extends ConsumerWidget implements PreferredSizeWidget {
               strokeWidth: 2,
               strokeCap: StrokeCap.round,
               valueColor: AlwaysStoppedAnimation<Color>(iconColor),
+              semanticsLabel: 'backup_controller_page_backup'.tr(),
```

Comment:
In the future, it would be cool if these labels included the status of the backup, since that's a cue for sighted users that screen readers miss. Something like:

* Backup (complete)
* Backup (in progress)
* Backup (off)